core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -339,6 +339,8 @@ public void cleanupSession(long sessionId) {
     }
     for (TempBlockMeta tempBlockMeta : tempBlocksToRemove) {
       try {
+        LOG.warn("Clean up expired temporary block {} from session {}.", tempBlockMeta.getBlockId(),
```

Comment:
This should only occur in rare cases which generally indicate something is wrong with the client-worker connection, so I'm fine with `WARN`